rules/warn-then-call/rule.md

```diff
@@ -0,0 +1,80 @@
+---
+type: rule
+title: Calling - Do you check your contact is free before calling them?
+uri: warn-then-call
+authors:
+  - title: Adam Cogan
+    url: https://ssw.com.au/people/adam-cogan
+  - title: Piers Sinclair
+     url: https://ssw.com.au/people/piers-sinclair
+  - title: Ulysses Maclaren
+     url: https://www.ssw.com.au/people/uly
+  - title: Jonty Gardner
+     url: https://www.ssw.com.au/people/jonty-gardner
+related:
+  - no-hello
+  - appointments-throw-it-in-their-calendar
+redirects:
+  - im-your-contact-to-see-if-they’re-free-before-you-try-calling-them
+  - calling-do-you-im-your-contact-to-see-if-theyre-free-before-you-try-calling-them
+  - calling-do-you-im-your-contact-to-see-if-they’re-free-before-you-try-calling-them
+  - im-your-contact-to-see-if-theyre-free-before-you-try-calling-them
+  - warn-then-try-calling-them
+created: 2015-10-05T05:50:19.000Z
+archivedreason: null
+guid: f1f96132-3b5d-43a6-9ed4-aed16f885875
+---
+It can be very jarring when somebody is called out of the blue and they are not expecting it. They might be deep in a task or talking to a client and by calling them their focus is getting disrupted.
```

Comment:
```suggestion
Before calling someone, [be sure not to just say 'hello'](/no-hello)

It can be very jarring when somebody is called out of the blue and they are not expecting it. They might be deep in a task or talking to a client and by calling them their focus is getting disrupted.
```